packages/messenger-widget/src/contexts/Accounts.ts

```diff
@@ -43,6 +43,14 @@ export function accountsReducer(state: Accounts, action: AccountsActions) {
                 contacts: action.payload,
             };
 
+        case AccountsType.Reset:
```

Comment:
What is the purpose of this functions ?

Reply:
In the current implementation, when the account changes the context API states do not get reset. So I added a reset action for every context states so that on account change we can use this action and reset entire context storage and it only resolved the issue of contacts and accounts header displaying the wrong details.